imports/startup/client/index.js

```diff
@@ -1,7 +1,3 @@
-
 import './routes'
-
-if (Meteor.isClient) {
```

Comment:
Are we sure it's safe to remove the check that it's on the client. Presumably it's ok because this file is only imported from /client?

Reply:
That's correct. This file is only included in the client/, no where else. The fact that it's also located in the "imports/startup/client" directory implies that you should not be implementing any server side logic here.

The larger issue actually has to do with the dynamic import statements. ES6 will try to get them to work, but it's not necessarily reliable:

"ES6 modules need to be able to statically resolve their dependencies, without executing module code, so that import statements do work reliably." https://stackoverflow.com/questions/30340005/importing-modules-using-es6-syntax-and-dynamic-path